manager/notifier.go

```diff
@@ -494,6 +563,15 @@ func (n *notifier) handleNotification(a *Alert, op notificationOp, config *pb.No
 			glog.Errorln("Error sending Slack notification:", err)
 		}
 	}
+	for _, fdConfig := range config.FlowdockConfig {
+		if op == notificationOpResolve && !fdConfig.GetSendResolved() {
+			continue
+		}
+		flowdockMessage := newFlowdockNotificationMessage(op, fdConfig, a)
+		url := *flowdockURL + "/" + fdConfig.GetApiToken()
+		httpResponse := postJSON(marshallJSON(flowdockMessage), url)
```

Comment:
Oh yeah, I didn't even notice that. I agree... also, the name would be `marshalJSON` (with one `l`) - see also `MarshalJSON` in the stdlib.